.github/workflows/generate-doc-check.yml

```diff
@@ -50,3 +50,6 @@ jobs:
 
       - name: Compile docs for all Scala versions
         run: sbt +compile:doc
+
+      - name: Compile paradox docs
+        run: sbt paradox
```

Comment:
do we need `sbt +compile:doc` if we do `sbt paradox`?

Reply:
Seems like it would be better to chain the commands.

Reply:
it's not a necessarily good idea to run separate commands. You will often find that the work is done from scratch, on each call.